uber/templates/registration/checkin.html

```diff
@@ -35,6 +35,7 @@
                 }
             });
             dialog.dialog('open');
+            dialog.dialog('option', 'position', {at: 'center', of: window});
```

Comment:
The check-in modal was showing up a bit too close to the top of the screen and getting squashed by the main admin menu.  So I made it centered on the page.